core/cloudflow-operator-actions/src/main/scala/cloudflow/operator/action/Name.scala

```diff
@@ -125,7 +125,7 @@ object Name {
     makeDNS1039Compatible(fixDots(streamletDeploymentName))
 
   def ofConfigMap(streamletDeploymentName: String) =
-    makeDNS1123CompatibleSubDomainName(s"configmap-${fixDots(streamletDeploymentName)}")
+    makeDNS1123CompatibleSubDomainName(s"configmap-${fixDots(streamletDeploymentName)}") // TODO - why append '-service' (line 145) but prepend 'configmap-'? - suffixes seem the more k8s-idiomatic choice
```

Comment:
just a rant - because the naming seem not too concise.

changing that, however, will likely introduce a whole lot of dead config-maps lying around on K8S.